README.md

```diff
@@ -82,4 +82,17 @@ On the control node run
  ansible-playbook playbook.yml -i inventory --vault-id tbs@prompt
 ```
 
-You will be prompted for the vault password
\ No newline at end of file
+You will be prompted for the vault password
+
+## Running the App using Docker
+
+To run on the dev environment use:
+
+```
+docker-compose up
+```
+
+To run on the prod environment use:
+```
+docker run -p 3000:3000 --env-file .env todo-app:prod
+```
```

Comment:
We're relying on the right image being build here, so it's probably worth explicitly calling out the `docker build --target production --tag todo-app:prod .` step as well

Reply:
Good point !